### DISCUSSION TITLE
can i implement it in Expo?

### DISCUSSION THREAD

**Discussion by LillyBrainy** [gre/gl-react-image#31] [Q&A] [answered]
yeah it's independent from this libary, gl-react-image is just a shader, the texture can be anything. it's up to the gl-react side to support video basically (and it already works in web)

some examples in https://gl-react-cookbook.surge.sh/blurvideo

_Originally posted by @gre in https://github.com/gre/gl-react-image/issues/30#issuecomment-635798835_

**Comment by gre** [comment, marked as answer]
Yes 👍
